source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -48,6 +48,8 @@ Configuration options for file integrity monitoring:
 alert_new_files
 ---------------
 
+.. deprecated:: 4.0
```

Comment:
It has not been deprecated because it is still working for older agents. I'd remove this comment.

Reply:
Done